sql-cli/include/base/config/default/configuration.yml

```diff
@@ -7,19 +7,15 @@ connections:
     schema:
     login:
     password:
-
 ## Example of Bigquery connection
-#  - conn_id: bigquery_conn
-#    conn_type: bigquery
-#    description: null
-#    extra:
-#      project: "my-gcp-project"
```

Comment:
Why did we remove the project from here? Should we adjust the `dev` configuration.yml as well?

Reply:
Because it currently does not work. See https://github.com/astronomer/astro-sdk/issues/1277

Reply:
Added a comment in the same ticket to check whether we need to update the dev configuration.yml too.